File: Execício 5 - Revisão Prática/decisao/decisao.cs
using System;

class decisao {
    static void Main() {
        Console.Write("Digite dois números inteiros\n");
        string[] parts = Console.ReadLine()?.Split();
        if (parts == null || parts.Length < 2) {
            Console.WriteLine("Entrada inválida.");
            return;
        }

        int a = int.Parse(parts[0]);
        int b = int.Parse(parts[1]);
        double media = (a + b) / 2.0;
        
        Console.WriteLine($"Média = {media}");
        if (b == 0) Console.WriteLine("b é zero (cuidado com divisão).");
        else if (media >= 10.0) Console.WriteLine("Média >= 10.0");
        else Console.WriteLine("Média < 10.0");
    }
}

File: Execício 5 - Revisão Prática/decisao/decisao.java
import java.util.Scanner;

public class decisao {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite dois números inteiros\n");

        if (!sc.hasNextInt()) { System.out.println("Entrada inválida."); return; }
        int a = sc.nextInt();
        if (!sc.hasNextInt()) { System.out.println("Entrada inválida."); return; }
        int b = sc.nextInt();

        double media = (a + b) / 2.0;
        System.out.println("Média = "+ a +"");
        if (b == 0) {
            System.out.println("b é zero.");
        } else if (media >= 10.0) {
            System.out.println("Média >= 10.0");
        } else {
            System.out.println("Média < 10.0");
        }

        sc.close();
    }
}

